qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -14,38 +14,38 @@
 
 public class ManageAppointmentsPage extends Page {
 
-	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
-	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons > button");
-	private static final By SEARCH = By.cssSelector("#searchButtons > button.confirm");
+	public static final By DELETE_REQUEST_ICON = By.cssSelector(".delete-item.icon-remove");
 	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By NEXT = By.cssSelector("#selectAppointment > button.confirm");
-	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
-	private static final By BOOK_APPOINTMENT = By.cssSelector("i.icon-calendar:nth-child(1)");
```

Comment:
Are these changes necessary.

Reply:
@sherrif10 yes all those files are involved in the appointment tests. The changes were made such that the variable names make more sense depending on what they are standing for.

Reply:
Are there more reviews on this?